hash/risc0/bench/run.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+if [ -z ${ZKBENCH_HASH_TYPE} ]; then 
+ZKBENCH_HASH_TYPE="sha256"    
+fi
+
+if [ -z ${ZKBENCH_WHICH} ]; then 
+ZKBENCH_WHICH="LINEAR"    
+fi
+
+if [ -z ${ZKBENCH_NTHREADS} ]; then 
+ZKBENCH_NTHREADS=1    
+fi
+
+if [ -z ${ZKBENCH_INPUT_SIZE_KB} ]; then 
+ZKBENCH_INPUT_SIZE_KB=1024
+fi
+
+if [ -z ${ZKBENCH_TREE_DEPTH} ]; then 
+ZKBENCH_TREE_DEPTH=4
```

Comment:
Currently, a single benchmark is supposed to have uniform parameters (though maybe we can relax that in the future, but providing more than one of the `bench.cfg` files). 

So right now it's not good that for some hashes you need the specify bytes or kilobytes  (btw, i think this is bytes, not kilobytes?), and for other the tree depth. One way out is to compute the tree depth from the bytes, but that a full tree cannot have any size. 

So maybe it would be better to have two `bench.cfg` files, and I wanted to add the support for simple formulas there, so that the benchmark framework can compute the number of bytes from the tree size instead (so that we can compare the speeds of byte-based and tree-based stuff, for example)